plugins/azure_ad_admin/requirements.txt

```diff
@@ -2,3 +2,4 @@
 # All dependencies must be version-pinned, eg. requests==1.2.0
 # See: https://pip.pypa.io/en/stable/user_guide/#requirements-files
 gunicorn==19.9.0
+parameterized==0.8.1
```

Comment:
@dsliwinski-r7 , would you mind updating the version of gunicorn to the latest please? I've noticed there is a snyk vulnerability with that version.

Reply:
@dlaverty-r7 I updated gunicorn to the latest version.